### PR TITLE
Smart Wallets: More Targeted Logging (Resolves WAL-2453)

### PR DESCRIPTION
**Pull request by devlyn37**
## Description

A few SDK versions ago we we're having a really hard time figuring out what was happening during issues from clients, and we ended up taking the nuclear approach for logs, logging the input and output of basically every single function.

This PR removes `LoggerWrapper`, the class responsible for that behavior, and drops a couple other small pieces. There's a few reasons to do this:
- Our current logs are way too noisy
- When on localhost, we log JWTs in the browser, which can make clients think we do so in other envs.

The key pieces that need strong logging, like when sending transactions, signing messages, etc are all covered by the `SmartAccountClient` decoration we do now.

## Test plan

Went through wallet fetching/creation and minting flow on the demo site and confirmed logs work as expected

### CODE REVIEW

File: packages/client/wallets/smart-wallet/src/utils/log.ts [hunk 1]
```diff
@@ -1,115 +1,15 @@
...
-    }
-}
 
 export async function logPerformance<T>(name: string, cb: () => Promise<T>, extraInfo?: object) {
     const start = new Date().getTime();
     const result = await cb();
     const durationInMs = new Date().getTime() - start;
     const args = { durationInMs, ...extraInfo };
-    logInfoIfNotInLocalhost(`[${SCW_SERVICE} - ${name} - TIME] - ${beautify(args)}`, { args });
+    logInfo(`[${SCW_SERVICE} - ${name} - TIME] - ${beautify(args)}`, { args });
```
Comment (devlyn37) [inline packages/client/wallets/smart-wallet/src/utils/log.ts:9]: Not needed

File: packages/client/wallets/smart-wallet/src/utils/log.ts [hunk 2]
```diff
@@ -1,115 +1,15 @@
...
-        return logPerformance(name, cb, this.extraInfo);
-    }
-}
 
 export async function logPerformance<T>(name: string, cb: () => Promise<T>, extraInfo?: object) {
     const start = new Date().getTime();
     const result = await cb();
     const durationInMs = new Date().getTime() - start;
     const args = { durationInMs, ...extraInfo };
-    logInfoIfNotInLocalhost(`[${SCW_SERVICE} - ${name} - TIME] - ${beautify(args)}`, { args });
```
Comment (devlyn37) [inline packages/client/wallets/smart-wallet/src/utils/log.ts:81]: not needed

Comment (jmderby) [reply packages/client/wallets/smart-wallet/src/utils/log.ts:81]: nice

**Comment by jmderby** [comment]
changes lgtm, logs still work in browser but are reduced :+1:

**Review by jmderby — APPROVED** [review]

**Pull request merged**